src/native-stack/views/NativeStackView.tsx

```diff
@@ -106,7 +107,16 @@ export default function NativeStackView({
               });
             }}>
             <HeaderConfig {...options} route={route} />
-            <View style={viewStyles}>{renderScene()}</View>
+            {__DEV__ &&
```

Comment:
__DEV__ checks can be stripped in build time but not when used like this. You can define a debug container in top level as follows and the use a simple if (__DEV__) statement:

```


let Container = View;

if (__DEV__) {
  const DebugContainer = (props) => <AppContainer><View {...props}/></AppContainer>
  Container = DebugContainer;
}
```